plugins/a11yhelp/dialogs/a11yhelp.js

```diff
@@ -103,18 +102,12 @@ CKEDITOR.dialog.add( 'a11yHelp', function( editor ) {
 	var variablesPattern = /\$\{(.*?)\}/g;
 
 	var replaceVariables = ( function() {
-		// Swaps keystrokes with their commands in object literal.
-		// This makes searching keystrokes by command much easier.
-		var keystrokesByCode = editor.keystrokeHandler.keystrokes,
-			keystrokesByName = {};
-
-		for ( var i in keystrokesByCode )
-			keystrokesByName[ keystrokesByCode[ i ] ] = i;
-
 		return function( match, name ) {
+			var keystrokeName = editor.getCommandKeystroke( name );
```

Comment:
If you look at [getCommandKeystroke](http://docs.ckeditor.com/#!/api/CKEDITOR.editor-method-getCommandKeystroke) API you'll see that it's returning a number. So it's a keystrokeCode not name.